csvqb/csvqb/tests/behaviour/qbwriter.feature

```diff
@@ -51,9 +51,60 @@ Feature: Test outputting CSV-Ws with Qb flavouring.
     When the cube is serialised to CSV-W
     Then csvlint validation of "duplicate-qube.csv-metadata.json" should succeed
-    # todo: complete me in Issue #65
+  # todo: complete me in Issue #65
 
   Scenario: A multi-measure QbCube with duplicate rows should fail validation
     Given a multi-measure QbCube named "Duplicate Qube" with duplicate rows
     When the cube is serialised to CSV-W
-    Then csvlint validation of "duplicate-qube.csv-metadata.json" should fail with "duplicate_key"
\ No newline at end of file
+    Then csvlint validation of "duplicate-qube.csv-metadata.json" should fail with "duplicate_key"
+
+  Scenario: A QbCube with string literal attributes should validate successfully
+    Given a single-measure QbCube named "Qube with string literals" with a "string" attribute
+    When the cube is serialised to CSV-W
+    Then csvlint validation of "qube-with-string-literals.csv-metadata.json" should succeed
+    And csv2rdf on all CSV-Ws should succeed
+    And turtle should be written to "output.ttl"
+    And the RDF should contain
+      """
+      <file:/tmp/qube-with-string-literals.csv#obs/uss-cerritos> <file:/tmp/qube-with-string-literals.csv#attribute/first-captain>
+      "William Riker".
+      """
+    And the RDF should contain
+      """
+      <file:/tmp/qube-with-string-literals.csv#obs/uss-titan> <file:/tmp/qube-with-string-literals.csv#attribute/first-captain>
+      "Carol Freeman".
+      """
+
+  Scenario: A QbCube with numeric literal attributes should validate successfully
+    Given a single-measure QbCube named "Qube with int literals" with a "int" attribute
+    When the cube is serialised to CSV-W
+    Then csvlint validation of "qube-with-int-literals.csv-metadata.json" should succeed
+    And csv2rdf on all CSV-Ws should succeed
+    # And turtle should be written to "output.ttl"
+    And the RDF should contain
+      """
+      <file:/tmp/qube-with-int-literals.csv#obs/uss-cerritos> <file:/tmp/qube-with-int-literals.csv#attribute/reg>
+      "75567"^^<http://www.w3.org/2001/XMLSchema#int>.
+      """
+    And the RDF should contain
+      """
+      <file:/tmp/qube-with-int-literals.csv#obs/uss-titan> <file:/tmp/qube-with-int-literals.csv#attribute/reg>
+      "80102"^^<http://www.w3.org/2001/XMLSchema#int>.
+      """
+
+  Scenario: A QbCube with date literal attributes should validate successfully
+    Given a single-measure QbCube named "Qube with date literals" with a "date" attribute
+    When the cube is serialised to CSV-W
+    Then csvlint validation of "qube-with-date-literals.csv-metadata.json" should succeed
+    And csv2rdf on all CSV-Ws should succeed
+    # And turtle should be written to "output.ttl"
+    And the RDF should contain
+      """
+      <file:/tmp/qube-with-date-literals.csv#obs/uss-cerritos> <file:/tmp/qube-with-date-literals.csv#attribute/appeared>
+      "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date>.
+      """
+    And the RDF should contain
+      """
```

Comment:
Good to see you've tested the common string/int/date attributes that we would want.

Have you tested an `ExistingQbAttributeLiteral` anywhere or are you just testing `NewQbAttributeLiteral`s?